RedditSharp/Things/Comment.cs

```diff
@@ -55,15 +55,16 @@ public Comment Init(Reddit reddit, JToken json, IWebAgent webAgent, Thing sender
         public Comment PopulateComments(IEnumerator<Thing> things)
         {
             Thing first = things.Current;
-            Dictionary<string, Tuple<Comment, List<Comment>>> comments = new Dictionary<string, Tuple<Comment, List<Comment>>>();
-            comments[this.FullName] = Tuple.Create<Comment, List<Comment>>(this, new List<Comment>());
-
+            Dictionary<string, Tuple<Comment, List<Comment>>> comments = new Dictionary<string, Tuple<Comment, List<Comment>>>
+            {
+                [this.FullName] = Tuple.Create(this, new List<Comment>())
+            };
             while (things.MoveNext() && (first is Comment || first is More))
             {
                 first = things.Current;
                 if (first is Comment)
                 {
-                    Comment comment = (Comment)first;
+                    var comment = (Comment)first;
```

Comment:
What's the reasoning for changing this to `var` from the more specific `Comment` ?